iterations_runners/iterations_runner.java

```diff
@@ -131,7 +131,10 @@ public void getInstJSON(StringBuilder sb, int iterNum) {
         System.loadLibrary("kruntime");
     }
 
+    static long UNINITIALISED_TSR = -1;  // All ones
```

Comment:
Not sure what you mean. long is signed, so -1 when interpreted unsigned is 2^64-1. This default value is always overwritten.